docs/content/en/docs/install/install.md

```diff
@@ -80,44 +74,79 @@ as well as for upgrades.
 Some helpful hints:
 
 * Use the `--version <version>` flag on the
-  `helm upgrade --install` command line to specify a different Keptn version.
+  `helm upgrade --install` command to install a Keptn version
+  other than the latest.
+  For example,
+
+  ```shell
+  helm upgrade --install keptn keptn/keptn \
+   --version version klt-v0.8.2 \
```

Comment:
For the --version flag, we need to provide the chart version number, which is not necessarily the same as the version of keptn (this is a bit confusing in helm). E.g. this would be the output of the `helm search repo keptn` command, where keptn v0.9.0 is installed via chart verison 0.3.0:

```
NAME                            CHART VERSION   APP VERSION     DESCRIPTION                                       
keptn/keptn                     0.3.0           v0.9.0          A Helm chart for Keptn Lifecycle Toolkit, a set...
keptn/keptn-cert-manager        0.2.0           v1.2.0          A Helm chart for Keptn Certificate Manager, a s...
keptn/keptn-lifecycle-operator  0.1.0           v0.8.3          A Helm chart for Keptn Lifecycle Operator, a su...
keptn/keptn-metrics-operator    0.1.0           v0.8.3          A Helm chart for Keptn Metrics Operator, a subp...
keptn/common                    0.1.0                           A Helm chart containing common templates for Keptn
keptn/klt                       0.2.6           v0.8.2          A Helm chart for Keptn Lifecycle Toolkit, a set...
```

```suggestion
   --version 0.3.0 \
```

Reply:
Ooh, this is ugly and definitely worthy of a bit of prose as well!  So they should use the chart version for keptn/keptn, right?  They don't need to worry about he fact that the subcharts have different versions?

But your sample only shows the keptn/keptn for the current "APP VERSION" (v0.9.0), right?  And my example showed how to install v.0.8.2, so is that the keptn/klt line, where the chart version is 0.2.6?

And, don't I need this information before I install Keptn, so would this be available on my system?

Maybe I should omit the sample output for the `helm search repo keptn` command and, instead, just create a little table that. shows the chart version for the previous releases that they can install rather than latest?  Either way, we're going to need to manually update this info for each release, right?

Reply:
yes, if all components of keptn should be installed, then the chart version 0.3.0 of `keptn/keptn` should be used. And yes, for installing keptn 0.8.2, that would be `keptn/klt`, chart version v0.2.6. The reason the search repo output only shows the latest version is because previously we used `keptn/klt` for the chart name, but now switched to `keptn/keptn`, so right now there is only one version of each chart. with future releases the command should show all available versions though.
As for the table, I would not add that as this would require constant maintenance whenever we release. I think it should be sufficient to refer to the `helm repo search` command to list all available versions, as this should be a well known concept for anyone using helm

Reply:
So are we saying that, right now, it's not possible to specify a release before 0.9.0 here?  Because of the switch to umbrella charts and all?

So you say "if all components of keptn should be installed"...  So are there times when I could specify, say, the chart version for and earlier release of the metrics-operator?  But then what version of cert-manager would I get or would I need to somehow specify two components?

And, if I am getting ready to install Keptn for the first time, would the `helm search repo keptn` command return anything?  I guess maybe this queries our official repo and not what is installed on your system?

Reply:
Yes, due to the switch to umbrella charts there is only 0.9.0 that can be installed this way.

It seems like the question about selecting which components should be installed is already answered in this PR.

The `helm search repo keptn` command will return something as soon as the `helm repo add` command has been executed, so yes, this queries the repo and not what is installed on your system

Reply:
So do we just say that it is now impossible to install anything before 0.9.0 or tell them to check the docs from earlier versions for the install line?  Of course, we weren't as explicit there -- would `--version 0.8.2` work with the install guide that references klt/klt?

Reply:
i'd say let's tell them to check the docs for earlier versions. Version 0.8.2 would be installed via `keptn/klt --version 0.2.6`

Reply:
After discussion in stand-up, I get it.  I added a very brief note to cover this.